apps/passport-server/src/routing/routes/telegramRoutes.ts

```diff
@@ -28,11 +28,12 @@ export function initTelegramRoutes(
    * Telegram.
    */
   app.get(
-    "/telegram/verify/:id/:username?",
+    "/telegram/verify/:chatId/:id/:username?",
```

Comment:
This will invalidate any old verify links. Ie, if you clicked start on ZuKat a week ago, that link won't work anymore. However, you can type /start` again to get a new link.